### PR TITLE
Adds a new scanner image to CCCP index

### PR DESCRIPTION
**Pull request by navidshaikh**
This PR adds a new container image to CCCP index. The image is an atomic scanner, namely `scanner-rpm-verify`.  It checks verifies and reports the problems with installed RPM packages and their files in given image under test.

### CODE REVIEW

**Comment by navidshaikh** [comment]
Depends on https://github.com/CentOS/container-pipeline-service/pull/46

File: index.yml [hunk 1]
```diff
@@ -530,3 +530,13 @@ Projects:
     target-file : Dockerfile.mailserv
     notify-email: bamachrn@gmail.com
     depends-on  : null 
+
+  - id          : 3004
+    app-id      : container-pipeline
```
Comment (dharmit) [inline index.yml]: This needs to be `pipeline-images`.

**Comment by navidshaikh** [comment]
I have updated the PR w.r.t to changes of index.d .

**Comment by kbsingh** [comment]
@dharmit are we good to merge this as yet, is there a corrosponding worker thread for using this scanner ?

**Comment by dharmit** [comment]
@kbsingh we do not have a worker thread using this scanner yet.

To use this scanner, we need to fix [pipeline service issue #48](https://github.com/CentOS/container-pipeline-service/issues/48) which has corresponding PR. But again, we need to take a decision on [pipeline service #66](https://github.com/CentOS/container-pipeline-service/issues/66) to make PR for [#48](https://github.com/CentOS/container-pipeline-service/issues/48) more usable.

**Pull request merged**